services/QuillLMS/client/app/bundles/Shared/styles/conceptFeedback.scss

```diff
@@ -1,3 +1,10 @@
+.control {
+  margin-top: 10px;
+}
+
+.control button {
+  margin-right: 10px;
+}
```

Comment:
This is going to change the CSS for anywhere we use the `.control` class -- maybe we can nest this rule inside of `.concept-explanation`?